project.clj

```diff
@@ -4,17 +4,22 @@
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
   :dependencies [[org.clojure/clojure "1.8.0"]
+                 [org.clojure/java.jdbc "0.6.2-alpha3"]
                  [http-kit "2.1.18"]
                  [bidi "2.0.11"]
                  [ring "1.5.0"]
                  [cheshire "5.6.3"]
-                 [jarohen/nomad "0.7.2"]
                  [honeysql "0.8.1"]
                  [ragtime "0.6.3"]
-                 [postgresql "9.3-1102.jdbc41"]]
+                 [postgresql "9.3-1102.jdbc41"]
+                 [ring/ring-json "0.4.0"]
+                 [com.mchange/c3p0 "0.9.5.2"]
```

Comment:
I actually just looked this up from [here](http://clojure-doc.org/articles/ecosystem/java_jdbc/connection_pooling.html). This seems easy to change later on so I won't worry about it for now.